### PR TITLE
Mat 4754 save elm json4 imported measure

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-4754](https://jira.cms.gov/browse/MAT-4754)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/mat/cql_elm_translation/controllers/MatMeasureController.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import gov.cms.mat.cql_elm_translation.controllers.CqlConversionController.TranslatorOptionsRemover;
+import gov.cms.mat.cql_elm_translation.data.RequestData;
+import gov.cms.mat.cql_elm_translation.service.CqlConversionService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@RequestMapping(path = "/mat/translator")
+@Tag(name = "Conversion-Controller", description = "API for converting MAT measure CQL to ELM.")
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/mat/cql_elm_translation/controllers/MatMeasureController.java]: "Conversion" is already heavily used by the community to indicate a measure model change: QDM to FHIR, FHIR to QICore, etc.

Could we go with "Transfer-Translator" or "Import-Translator" Controller, instead?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/mat/cql_elm_translation/controllers/MatMeasureController.java]: @jkotanchik-SB Ok, sure.

**Review by jkotanchik-SB — APPROVED** [review]

**Review by adongare — APPROVED** [review]
